modules/azure/frontdoor.go

```diff
@@ -0,0 +1,127 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/frontdoor/mgmt/frontdoor"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// FrontDoorInstanceExists indicates whether the Front Door instance exists for the subscription.
+// This function would fail the test if there is an error.
```

Comment:
NIT: Remove "Instance" from the all method names in this file , eg . "FrontDoorInstanceExists" to be "FrontDoorExists"